### ISSUE TITLE
Bitbucket: Download failed. Too many redirects

### ISSUE THREAD

**Issue by mobiusint** [afragen/git-updater#1067]
Hi again,
Thanks for your awesome plugin. I am trying to connect bitbucket to my WP install and keep getting this error. 
"Download failed. Too many redirects"

I have tried using my personal bitbucket password & the app password as indicated in the knowledge base. Which password should I be using? I have tried both with the same error. 

I also checked php error log and there aren't any errors.

The main screen for bitbucket where I entered app password doesn't show anything as well. I'm not sure if that's correct or not.
Screenshot: https://capture.dropbox.com/gPkJp8pxFnzR3c6y

I check the wp_options field and the app password is in there correctly with the ":".

Thank you in advance
:)

**Comment by afragen** [comment]
The Knowledge Base has info and a link about the application password. 

https://git-updater.com/knowledge-base/settings/#articleTOC_5

What are you doing when you get the error?

Is the repo public? Can you share the URL?

**Comment by mobiusint** [comment]
Hi there,
Thanks for quick reply. I read from issues it has to do with password area, but I am 100% sure I have put password correctly.

The knowlegebase is where I read about application password. I have tried with my email and regular passsword & application password. Neither work.

I have tried installing theme & plugin from same bitbucket account. 
It is a private repository.

Here is screenshot: https://capture.dropbox.com/blE8dQYgNftNy7ZI

Error occurs after i click install plugin.
